File: 2st/Riyeon/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] arr = s.split(" ", -1);
        for(int i=0; i<arr.length; i++){
            if(i > 0) answer += " ";
            String str = arr[i];
            for(int j=0; j<str.length(); j++){
                char ch = str.charAt(j);
                if(j%2==0)
                    answer += Character.toUpperCase(ch);
                else
                    answer += Character.toLowerCase(ch);
            }
        }
        return answer;
    }
}

File: 2st/Riyeon/이상한 문자 만들기.py
def solution(s):
    answer = []
    for li in s.split(' '):
        st = ''
        for i in range(len(li)):
            st += li[i].upper() if i%2 == 0 else li[i].lower()
        answer.append(st)
    return ' '.join(answer)